### PR TITLE
Validate father in same case as child

### PR DESCRIPTION
**Pull request by seallard**

### CODE REVIEW

File: cg/services/order_validation_service/models/errors.py [hunk 1]
```diff
@@ -71,5 +71,10 @@ class RepeatedSampleNameError(CaseSampleError):
 
 
 class InvalidFatherSexError(CaseSampleError):
-    field: str = "sex"
+    field: str = "father"
     message: str = "Father must be male"
+
+
+class InvalidFatherCaseError(CaseSampleError):
```
Comment (islean) [inline cg/services/order_validation_service/models/errors.py:78]: Slightly unclear naming but probably workable. I would suggest `FatherNotInCaseError`.

**Review by islean — APPROVED** [review]
Looks good to me! Sometime soon it might be nice to start writing tests that check multiple validations? I think your tests should have covered for the case when a father is missing - If so, we do not want has failed the sex check as well but we want this PR's validation to give errors. 

Not the best example since I think your unit tests are sufficient for it, but something to consider.

File: cg/services/order_validation_service/models/errors.py (hunk 1 above)
Comment (seallard) [reply cg/services/order_validation_service/models/errors.py]: Nice, changed it https://github.com/Clinical-Genomics/cg/pull/3492/commits/59c55d28cdd64bd4530f77b20ba88e7aa2f3cfda

**Comment by seallard** [comment]
> Looks good to me! Sometime soon it might be nice to start writing tests that check multiple validations? I think your tests should have covered for the case when a father is missing - If so, we do not want has failed the sex check as well but we want this PR's validation to give errors.
> 
> Not the best example since I think your unit tests are sufficient for it, but something to consider.

Good point! I'll add unit tests for this rule as well. Maybe we can hold off on integration tests until we have the entire tomte flow in place.

**Pull request merged**